jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/WebAppContextTest.java

```diff
@@ -113,6 +119,36 @@ private Server newServer()
         return server;
     }
 
+    /**
+     * Create a webapp as a war on the fly.
+     *
+     * @param tempDir the directory into which the war will be generated
+     * @param name the name of the war
+     * @return the Path of the generated war
+     *
+     * @throws Exception
+     */
+    private Path createWar(Path tempDir, String name) throws Exception
+    {
+        // Create war on the fly
+        Path testWebappDir = MavenPaths.projectBase().resolve("src/test/webapp");
```

Comment:
The `src/test/webapp` tree doesn't exist in the ee8 tree.

Move `src/test/webapp` contents to `src/test/resources/webapp-foo/` first.
Then use this line instead ...

``` java
Path testWebappDir = MavenPaths.findTestResourceDir("webapp-foo");
```

Reply:
Fixed! Thanks @joakime for the hint!